how-to/releaseduty/merge-duty/merge_duty.rst

```diff
@@ -21,29 +21,28 @@ nearly three weeks, with *three* major days of activity:
 
 -  Do the prep work a week before the merge
 
-   -  `Assign to yourself the migration bug <#assign-to-yourself-the-migration-bug>`__
-   -  `Do migration no-op trial runs <#do-migration-no-op-trial-runs>`__
+   -  `Assign to yourself the migration bug <#assign-to-yourself-the-migration-bug>`__ (1 min)
```

Comment:
We've stopped filing migration bugs a while back.  Should we remove it from here or start using them again?

Reply:
Let's remove them, then